examples/basic/spago.dhall

```diff
@@ -0,0 +1,15 @@
+let config = ./../../spago.dhall
+
+in      config
+    //  { sources = config.sources # [ "examples/basic/**/*.purs" ]
+        , dependencies =
+              config.dependencies
+            # [ "arrays"
+              , "exceptions"
+              , "foldable-traversable"
+              , "integers"
+              , "react-basic"
+              , "react-basic-dom"
+              , "web-html"
+              ]
+        }
```

Comment:
This formatting looks a bit strange, but this is how `dhall format` likes things 🤷